### PR TITLE
Set swig to use relative paths and just use a single search path

### PR DESCRIPTION
**Pull request by Dekken**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -289,15 +289,13 @@ def add_dir_name(dir_name, filenames):
...
-    for opts in [swig_opts, extra_compile_args]:
-        opts.extend(["-Isrc/swig/"+extension_name])
 
     libraries = []
     library_dirs = []
     runtime_library_dirs = []
     extra_link_args = []
     define_macros = []
-    extra_include_dirs = ["include"]
+    extra_include_dirs = ["include", "swig"]
```
Comment (Mbompr) [inline setup.py]: This avoids using `opts.extend(["-Isrc/swig/"+extension_name])` ?

Comment (Dekken) [reply setup.py]: indeed, we don't need to add each module directory as an include any longer

**Comment by Dekken** [comment]
Yes indeed, thanks for that, I should have put it in the first comment!

**Comment by Mbompr** [comment]
Cool, LGTM :)

**Pull request merged**